src/plugins/kibana/public/settings/sections/objects/_objects.js

```diff
@@ -93,13 +93,11 @@ uiModules.get('apps/settings')
         retrieveAndExportDocs(objs);
       };
 
-      $scope.exportAll = () => {
-        Promise.map($scope.services, (service) =>
-          service.service.scanAll('').then((results) =>
-            results.hits.map((hit) => _.extend(hit, {type: service.type}))
-          )
-        ).then((results) => retrieveAndExportDocs(_.flattenDeep(results)));
-      };
+      $scope.exportAll = () => Promise.map($scope.services, service => service.service
+        .scanAll('')
+        .then(results => results.hits.map(hit => _.extend(hit, { type: service.type })))
+        .catch(error => notify.error(error))
```

Comment:
This actually results in three different error blocks in the "more info" section of the error banner.  I assume because there are three services or something, though I'm not familiar enough with this feature to say for certain.  Perhaps the notify behavior should be moved to a .catch() on the .map() promise instead?